git-players.text

```diff
@@ -1,5 +1,5 @@
 <<record
-player_name: THS
+player_name: MTS
```

Comment:
Hey can you add a new record? Like this

```
<<record
player_name: new name
player_display_name: new display name
player_github: etc
player_favorite_thing: etc
player_public_email: etc
```

Reply:
Done!

Reply:
"Add a player record to the table"

```
<<record
player_name: THS
player_display_name: THSPower
player_github: https://github.com/thspower
player_favorite_thing: Saving the world
player_public_email: hspowersport@gmail.com
<<record
player_name: MTS
player_display_name: MTSEnergy
player_github: https://github.com/thspower
player_favorite_thing: The pursuit of excellence
player_public_email: silva.friends22@gmail.com
```

Reply:
New record added: Player's favorite game.

Reply:
Like this

https://github.com/thspower/free-energy/pull/2/files

😅 Add a second player to the file

Reply:
Now yes lol